### DISCUSSION TITLE
Need help on "ca md too weak" on the given setup command for the test server

### DISCUSSION THREAD

**Discussion by ManasPatil0967** [open-quantum-safe/liboqs#1978] [Q&A] [answered]
I am just getting started in the PQC space and wanted to setup a openssl test server with the pqc algorithms and came across liboqs and oqs-provider. I setup openssl, liboqs and oqs-provider as given.
OpenSSL version - OpenSSL 3.0.14 4 Jun 2024
I used all commands as asked in [Creating keys and certificates](https://github.com/open-quantum-safe/oqs-provider/blob/main/USAGE.md#creating-keys-and-certificates) and [Setting up a (quantum-safe) test server](https://github.com/open-quantum-safe/oqs-provider/blob/main/USAGE.md#setting-up-a-quantum-safe-test-server) and got the error "ca md too weak". I have even used the flag -sha256 in generating the keys. Can someone help me troubleshoot this?

Screenshot:
![Screenshot_20241105_112330](https://github.com/user-attachments/assets/6f3b3625-c5e6-429a-9755-edd3fb991625)

**Comment by baentsch** [comment, marked as answer]
FAQ, see e.g., https://github.com/open-quantum-safe/oqs-provider/discussions/455

**Reply by ManasPatil0967** [reply]
Thank you so much! That was poor look-up on my part.

**Reply by baentsch** [reply]
NP. UW!
